src/main/java/countdownlatch/D_zadanieCountDownLatch/Test.java

```diff
@@ -0,0 +1,37 @@
+package countdownlatch.D_zadanieCountDownLatch;
+
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * @author Kacper Staszek
+ * @author Marcin Ogorzałek
+ */
+class Test implements Runnable {
```

Comment:
Może detal, może późna pora, ale na początku szukałem "czego to test" i "co testujemy". Dopiero kilka sekund później uświadomiłem sobie, że to ten historyczny test. :-)